src/Umbraco.PublishedCache.NuCache/Property.cs

```diff
@@ -33,7 +33,7 @@ internal class Property : PublishedPropertyBase
     private object? _interValue;
 
     // the variant source and inter values
-    private readonly Lazy<ConcurrentDictionary<CompositeStringStringKey, SourceInterValue>> _sourceValues = new();
+    private readonly Lazy<ConcurrentDictionary<CompositeStringStringKey, SourceInterValue>> _sourceValues = new(() => new(-1, 5));
```

Comment:
I think this should be set depending on the expected number of entries, what's common?

Reply:
Hey @Nuklon,

This SO sums up [the choice here](https://stackoverflow.com/questions/60005428/concurrentdictionary-proper-small-initial-capacity)... 

> If you want to
>
>- optimize speed: take an initial capacity which is a prime about 30% bigger than the expected maximum dictionary size. This avoids resizing.
>- optimize the memory footprint: take a prime which is about 30% bigger than the minimum expected size.
>- a balance between speed and memory footprint: Take a number in between the two from above. But in any case, take a prime.

I'll double check the most common number, but it's _low, as it's effectively number of languages x variations so 5 seemed pretty sensible here.